sdk/android/build.gradle

```diff
@@ -13,6 +13,7 @@ buildscript {
 
 allprojects {
     repositories {
+        maven { url "https://maven.google.com" }
```

Comment:
why do we need it? Isn't `google()` the same as this?

Reply:
From google documentation: 

```
 If you're using a version of Gradle lower than 4.1, you must instead use:
 // maven {
 //     url 'https://maven.google.com'
 // }
```